src/System.IO.Compression.ZipFile/ref/System.IO.Compression.ZipFile.csproj

```diff
@@ -8,10 +8,19 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net471-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net471-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Release|AnyCPU'" />
   <ItemGroup>
-    <Compile Include="System.IO.Compression.ZipFile.cs" />
+    <Compile Include="System.IO.Compression.ZipFile.cs" Condition="'$(TargetGroup)' != 'net471' And '$(TargetGroup)' != 'netfx'" />
```

Comment:
Might be a little easier to understand if you just move these compile items under the ItemGroups with the matching references.

Reply:
sounds good. I'll change it.